AdminWebsite/AdminWebsite.UnitTests/Controllers/HearingsController/GetStatusHearingTests.cs

```diff
@@ -296,59 +295,6 @@ public async Task Should_return_ok_status_with_success_when_V2flag_on()
 
         }
 
-        [TestCase(false)]
-        [TestCase(true)]
-        public async Task Should_return_ok_status_with_success_when_users_not_created_for_successful_multi_day_booking_with_notify_flag_on(bool v2FlagOn)
```

Comment:
please can we keep this test, but remove the non-V2 path

Reply:
done